packages/block-library/src/image/edit.native.js

```diff
@@ -288,11 +278,11 @@ class ImageEdit extends React.Component {
 					{ ( ! this.state.isCaptionSelected ) &&
 						getToolbarEditButton( openMediaOptions )
 					}
-					<InspectorControls>
+					<InspectorControls buttons>
```

Comment:
tell Inspector Controls to be render in `InspectorControls.SlotButtons` instead of `InspectorControls.Slot`